### PR TITLE
fix #278977 Allow stripes to be highlighted in piano roll editor

### PR DESCRIPTION
**Pull request by blackears**
Can now control-click on keyboard area to toggle highlights on background bars.

### CODE REVIEW

File: mscore/preferences.h [hunk 1]
```diff
@@ -188,6 +188,7 @@ enum class MusicxmlExportBreaks : char {
 #define PREF_UI_PIANOROLL_DARK_NOTE_UNSEL_COLOR             "ui/pianoroll/dark/note/unselected/color"
 #define PREF_UI_PIANOROLL_DARK_NOTE_SEL_COLOR               "ui/pianoroll/dark/note/selected/color"
 #define PREF_UI_PIANOROLL_DARK_BG_BASE_COLOR                "ui/pianoroll/dark/background/base/color"
+#define PREF_UI_PIANOROLL_DARK_BG_KEY_HILIGHT_COLOR         "ui/pianoroll/dark/background/keys/hilight/color"
```
Comment (anatoly-os) [inline mscore/preferences.h]: typo? highlight?

Comment (blackears) [reply mscore/preferences.h]: I've seen it spelled this way sometimes asa a short form.

Comment (blackears) [reply mscore/preferences.h]: I've changed the bit that's visible to users.

**Comment by dmitrio95** [comment]
Rebase is needed here

**Comment by blackears** [comment]
Started a new PR:
https://github.com/musescore/MuseScore/pull/5807

**Comment by Jojo-Schmitz** [comment]
Sp can this here get closed then?

**Pull request closed without merging**